### PR TITLE
ci_vms_standalone_ptp.yaml: add vm ptp clock synchronisation in the CI

### PR DESCRIPTION
**Pull request by bchedotel**
Synchronize the system clock of each vms with phc2sys.

### CODE REVIEW

File: src/phc2sys.service [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Synchronize system clock or PTP hardware clock (PHC)
+Documentation=man:phc2sys
+After=ptp4l@%i.service
```
Comment (dupremathieu) [inline src/phc2sys.service:4]: You can not use the %i parameter with a regular service.
Create a template service: phc2sys@.service
```
[Unit]
Description=Synchronize system clock or PTP hardware clock (PHC) with %i
Documentation=man:phc2sys
Requires=ptp4l@%i.service
After=ptp4l@%i.service
Before=time-sync.target
```

**Review by dupremathieu — DISMISSED** [review]

File: src/phc2sys.service (hunk 1 above)
Comment (eroussy) [inline src/phc2sys.service:4]: ptp4l is simply not needed here, because we synchronise the VMs with ptp_kvm

**Review by eroussy — CHANGES_REQUESTED** [review]

File: playbooks/ci_vms_standalone_ptp.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# SPDX-Licence-Identifier: Apache-2.0
+
+# Ansible playbook that deploy and set-up ptp on vms for a standalone machine
+
+- import_playbook: deploy_vms_standalone.yaml
+
+- name: Check if phc2sys service is already started
+  hosts: VMs
+  tasks:
+    - name: Gather facts about services
```
Comment (eroussy) [inline playbooks/ci_vms_standalone_ptp.yaml:11]: I think you can just write 
```
    - name: Ensure phc2sys service is stopped
      systemd:
        name: phc2sys
        state: stopped
```
no need to gather facts are to type a when line

**Review by eroussy — CHANGES_REQUESTED** [review]

File: src/phc2sys.service (hunk 1 above)
Comment (bchedotel) [reply src/phc2sys.service]: I forget to remove the After, we do not use ptp4l because we use ptp_kvm inside the vm

**Review by eroussy — APPROVED** [review]

**Pull request merged**